.github/workflows/dart_mcp_server.yaml

```diff
@@ -55,3 +55,16 @@ jobs:
         if: ${{ matrix.flutterSdk == 'master' }}
 
       - run: dart test
+
+  readme:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - uses: dart-lang/setup-dart@e51d8e571e22473a2ddebf0ef8a2123f0ab2c02c
+
+      - run: dart pub get
```

Comment:
You could just merge this into the job above to avoid the extra setup work 🤷‍♂️ . This repo isn't large enough to really need multiple jobs.

Reply:
I kept this separate since this above job was doing more than we need here (using the flutter sdk instead of the dart sdk, ...).

Reply:
Gotcha - I merged this into the job above.